test/functional/legacy/cdo_spec.lua

```diff
@@ -108,6 +109,7 @@ describe('cdo', function()
         if subst_count != 1 || getline('.') != 'xLine1'
           call add(v:errors, 'Abort command on error test failed')
         endif
```

Comment:
The test in lines 106-111 fails if 'hidden' is set, but I'm not sure why. This is the only test where I said "eff it" and just used `set nohidden` to make it pass.